interface/usergroup/facility_user.php

```diff
@@ -80,6 +81,7 @@ function refreshme() {
         });
     </script>
 </head>
+
 <body class="body_top">
     <?php
     // Collect all users
```

Comment:
Hi @yashrajbothra 

Please help remove the `body_top `class it is unnecessary.